docs/src/contributing-to-popochiu/contributing-documentation/basic-rules.md

```diff
@@ -0,0 +1,87 @@
+
+!!! tip "Updating visual assets"
+    When the editor interface changes or get some new feature, screenshots or other visuals may need to be updated.  
+    Taking consistent screenshots is a tough job: we recommend using the **Popochiu Sample Game**. Clone the repo and update the `addons/popochiu` folder with your changes.
+
+    Remember to annotate the screenshots if it adds to clarity and readability of your explanation. If possible use Godot dark theme and red annotations for consistency.
+
+---
+
+## Understanding Documentation Types
+
+Before you start writing, it's important to understand what type of documentation you’re contributing to. Popochiu’s documentation broadly follows the categories defined by the [Diátaxis framework](https://diataxis.fr/), by which content fall into one of four types (each with a specific _user's main goal_ in brackets):
+
+- **Tutorials**: Step-by-step guides that show how stuff work (**learning**)
+- **How-to Guides**: Focused, task-oriented instructions to achieve a certain goal . think recipes (**achieving a goal**).  
+- **Explanations**: Conceptual overviews and background information (main goal: **understanding**)
+- **Reference**: Well-organized list of techical details and descriptions (main goal: **finding information**)
+
+### Documentation Sections and Content Types
+
+Each section in Popochiu's documentation aligns with a type from the Diátaxis framework. Here's a quick overview:
+
+| **Section**              | **Content types**        | **Purpose**                                                                                                               |
+|--------------------------|--------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| Getting Started          | Tutorials, Explanations  | Provides the basics for installing Popochiu, taking first steps, and joining the community.                               |
+| The Editor Handbook      | References, Tutorials    | Showcases the functionality of the Popochiu plugin, explains how it works, and serves as a go-to guide for its use.       |
+| The Engine Handbook      | References, Explanations | Explains the plugin's architecture and provides a comprehensive API reference for game developers.                        |
+| How to Develop a Game    | Tutorials, How-To        | Guides users through setting up their first game, learning by doing, and building confidence to explore advanced features. |
+| Advanced Techniques      | Tutorials, How-To        | Offers in-depth guides and practical examples for getting the most out of Popochiu (and beyond).                          |
```

Comment:
Based on this, one would think that the **Advanced Techniques** section should appear in the main navigation panel, but right now it’s a subcategory of the **How to Develop a Game** section. I’m not sure if this needs to be clarified or if the navigation panel needs to be updated.

Reply:
Ah! I'm an idiot! You're right, I totally forgot to reorganize the menu.

Thanks!